ui/src/stylesheets/article.scss

```diff
@@ -61,6 +63,14 @@
       color: #4f4f68;
     }
   }
+
+  // .ulist a {
```

Comment:
Can we remove the commented lines please?

Reply:
Fixed in 6f3548f